routes/roadmapPlan.js

```diff
@@ -9,25 +12,97 @@ const router = express.Router();
 // based on that plan
 router.post("/api/roadmapPlans/createFromPlan", (req, res) => {
   const old_id = req.body.id;
+  const desc = req.body.desc;
   plans
     .findById(old_id)
-    .then((retrieved) => {
+    .then(async (retrieved) => {
+      // extract simple fields from the plan
       let data = {
         original: retrieved.id,
         name: retrieved.name,
+        description: desc,
+        num_likes: 0,
```

Comment:
another option is to change the Roadmap schema to have a default value for this field